cli/ballerina-tool/src/main/resources/cli-help/ballerina-openapi-gen-client.help

```diff
@@ -1,36 +1,36 @@
 NAME
-       ballerina openapi gen-client - generate a mock Ballerina client
+       ballerina-openapi-gen-client - Generate a mock Ballerina client
        for a given OpenAPI contract
 
 
 SYNOPSIS
-       ballerina openapi gen-client [<module>:]<client-name>
-                                    <openapi-file> [-o=<outputFileName>]
+       ballerina openapi gen-client [<module-name>:]<client-name>
+                        <openapi-contract> [-o | --output <dir-path>]
 
 
 DESCRIPTION
-       Note: This is an experimental tool which only supports a
-       limited set of functionality.
+       Gen-client generates a Ballerina client for the provided OpenAPI
+       contract in the optionally given directory path. An optional module
```

Comment:
Shall we just say

"Gen-client generates a Ballerina client for the provided OpenAPI contract"